sdk/tables/Microsoft.Azure.WebJobs.Extensions.Tables/src/TablesExtensionConfigProvider.cs

```diff
@@ -120,6 +122,15 @@ private async Task<TableClient> CreateTableClient(TableAttribute attribute, Canc
             return table;
         }
 
+        internal Task<ParameterBindingData> CreateParameterBindingData(TableAttribute attribute, ValueBindingContext context)
```

Comment:
This might not need to be a task as we're not doing anything async? Have you tried removing Task and see if it still works? (Will also need to replace `Task.FromResult(parameterBindingData);` with `parameterBindingData`)

Reply:
I think we need it to be a task in order to bind to input

Reply:
It breaks when you remove Task?

Reply:
Yeah it breaks since `BindToInput` requires a task in the parameter

Reply:
There's a sync overload on BindToInput though - we're using it for other extensions, e.g. [here](https://github.com/Azure/azure-sdk-for-net/blob/8c54ddcf82e3ec630b302896f0e9d0dc2ccd39db/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Blobs/src/Config/BlobsExtensionConfigProvider.cs#L113) in blobs. Can't you use that?

Reply:
This is the error that I get when I change CreateParmaterBindingData to sync
<img width="559" alt="image" src="https://github.com/Azure/azure-sdk-for-net/assets/37918412/313b9954-05a6-4321-b471-e796ddbd5984">
`Argument 1 cannot convert from method group to Microsoft.Webjobs.IConverter<TableAttribute, ParameterBindingData>`

Is there a way to get around this error without converting it to a task?

Reply:
Ah I realized my issue was that I was passing on context even though it wasn't needed. Will convert the method to synchronous